packages/labs/ssr-dom-shim/src/index.ts

```diff
@@ -59,6 +59,10 @@ const ElementShim = class Element {
     }
     return shadowRoot;
   }
+  attachInternals(): ElementInternals {
+    const internals = new InternalsShim(this);
```

Comment:
Let's assign to a field like `this.__internals` to make available to other SSR code like ElementRenderers.